src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -95,18 +103,44 @@ public function getItem(string $resourceClass, $id, string $operationName = null
         $repository = $manager->getRepository($resourceClass);
         $queryBuilder = $repository->createQueryBuilder('o');
 
-        foreach ($identifiers as $propertyName => $value) {
-            $placeholder = 'id_'.$propertyName;
-
-            $queryBuilder
-                ->where($queryBuilder->expr()->eq('o.'.$propertyName, ':'.$placeholder))
-                ->setParameter($placeholder, $value);
-        }
+        $this->processIdentifiers($identifiers, $queryBuilder);
 
         foreach ($this->itemExtensions as $extension) {
             $extension->applyToItem($queryBuilder, $resourceClass, $identifiers, $operationName);
         }
 
         return $queryBuilder->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * Add where into the query when multiple identifiers are passed (composite).
+     *
+     * @param array        $identifiers
+     * @param QueryBuilder $queryBuilder
+     *
+     * Populate the query with where when needed.
+     */
+    private function processIdentifiers(array $identifiers, QueryBuilder $queryBuilder)
```

Comment:
What is `processIdentifiers` supposed to do? The name is not descriptive enough. Perhaps `addWhereForIdentifiers`?